src/metax_api/services/rems_service.py

```diff
@@ -143,7 +219,29 @@ def _post_rems(self, entity, body):
         resp = response.json()
 
         if not resp['success']:
-            raise REMSException(f'Could not create {entity} to REMS. Error: {resp["errors"]}')
+            raise REMSException(f'Could not {action} {entity} to REMS. Error: {resp["errors"]}')
+
+        return resp
+
+    def _put_rems(self, entity, action, body):
+        """
+        Edit rems entity. Possible actions: [edit, archived, enabled].
+        """
+        try:
+            response = requests.put(f"{self.base_url}/{entity}s/{action}", json=body, headers=self.headers)
+
+        except Exception as e:
+            _logger.error(f'Connection to REMS failed while updating {entity}. Error: {e}')
+            raise Exception(e)
+
+        if response.status_code != 200:
+            raise REMSException(f'REMS returned bad status while updating {entity}. Error: {response.text}')
```

Comment:
I assume these exception messages do not get propagated all the way to the client as error messages?